refs/types.proto

```diff
@@ -12,8 +12,14 @@ option (gogoproto.goproto_stringer_all) = false;
 
 // Address of object (container id + object id)
 message Address {
-    // ObjectID is an object identifier, valid UUIDv4 represented in bytes
-    bytes ObjectID = 1[(gogoproto.customtype) = "ObjectID", (gogoproto.nullable) = false];
+    // ObjectID carries object identifier.
+    ObjectID ObjectID = 1;
```

Comment:
Should we change OID/CID order in address? Having CID first might be more understandable.

Reply:
Yes, I planned to switch the field order in the coming API changes.